docs/UserGuide.md

```diff
@@ -4,339 +4,1044 @@
 
-**Format**: `find [n/SEARCH_STRING_COMPANY] [r/SEARCH_STRING_ROLE] [c/SEARCH_STRING_CYCLE] [s/SEARCH_STRING_STATUS] [d/SEARCH_STRING_DEADLINE][i/SEARCH_STRING_NOTE]`
+It enables you to seamlessly **input new applications** into your record. By keeping all your applications up-to-date, 
+you can efficiently keep track of your progress, upcoming interviews, and crucial deadlines.
 
-**Parameters**:
-* At least one of the optional fields must be provided.
+#### Command Usage
 
-**Examples**:
-* `find n/Google` displays all internship applications whose company name contains `Google`
-* `find n/Apl` displays all internship applications whose company name contains `Apl`
-  * These include `Apple`, `apple` and `Alphabet Limited`.
-* `find c/Summer` displays all internship applications whose cycle contains `Summer`
-* `find n/Google c/Summer s/Pending` displays all internship applications which meets at least one of the following conditions:
-  * Company name contains `Google`
-  * Cycle contains `Summer`
-  * Status is `Pending`
+**Command**:
+```
+add n/COMPANY_NAME r/ROLE c/CYCLE [s/STATUS] [d/DEADLINE]
+```
 
-**Expected Output**:
-![find command example](images/findExample.jpg)
+#### Parameters:
+
+1. **n/COMPANY_NAME**: The company's name where you've applied.
+   * Example: `Jane Street`, `Google`
+
+2. **r/ROLE**: The role you've applied for.
+    - Example: `Software Engineering Intern`
+
+3. **c/CYCLE**: The internship cycle, which typically corresponds to the season or term.
+    - Example: `Summer`, `Winter`
+
+4. **s/STATUS** (Optional): The current status of your application.
+   <box type="info" seamless>
+
+   Choose your application `STATUS` from the following:
+    - `Pending`
+    - `Interview`
+    - `Assessment`
+    - `Offered`
+    - `Rejected`
+
+    </box>
+
+5. **d/DEADLINE** (Optional): The application deadline or any other crucial dates.
+   <box type="info" seamless>
+
+   The `DEADLINE` should adhere to the `dd MMM yyyy` format. For instance: `25 Oct 2023`.
+   </box>
+
+#### Example:
+
+To add an internship application for a **Software Engineering** role at **Google** for the **Summer 2024** cycle with an **Offered** 
+status and a deadline on **25th October 2023**, you can type:
+```
+add n/Google r/Software Engineering c/Summer 2024 s/Offered d/25 Oct 2023
+```
+
+**_UPDATE PICTURE HERE_**
+
+![add command example 1](images/addExample1.jpg)
+
+By executing the `add` command, your application details are recorded and presented in the list. 
+This ensures that as you apply to more roles and receive updates on your applications, every piece of information is 
+systematically stored, making tracking and follow-ups more manageable.
+
+<box type="tip" light>
+
+* Regularly adding applications immediately after submission ensures you don't miss out on tracking any application.
+* Remember to update the `STATUS` of your applications as they progress. This way, you'll always be aware of which stage each application is at.
```

Comment:
Maybe we can tell the users somewhere that the default status is pending

Reply:
Got it! I missed this!